lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -609,7 +609,11 @@ defmodule ExUnit.Assertions do
       assert_in_delta 10, 15, 4
 
   """
-  def assert_in_delta(value1, value2, delta, message \\ nil) do
+  def assert_in_delta(value1, value2, delta, message \\ nil)
+  def assert_in_delta(_, _, delta, _) when delta < 0 do
+    raise ArgumentError, "all deltas must be positive. Did you mean #{abs(delta)}?"
```

Comment:
We can use `-delta` here instead of `abs(delta)` since we already know that `delta` is negative.